packages/office-ui-fabric-react/src/components/ContextualMenu/__snapshots__/ContextualMenu.test.tsx.snap

```diff
@@ -4,7 +4,7 @@ exports[`ContextualMenu ContextualMenu snapshot ContextualMenu should be present
 <button
   aria-expanded={false}
   aria-haspopup={true}
-  aria-owns="id__94-menu"
+  aria-owns={null}
```

Comment:
Is this change expected? I don't see any changes to the test, so this looks like a regression.

Also, this seems to be exposing a pre-existing issue in button. Unused ARIA attributes should be `undefined`, not `null`.

Reply:
It is not completely ok for this to change. But, I think it should be fine, This snapshot is taken on a menu which is persisted but hidden. In this case, the menu is not shown to the user, but present in the DOM with visibility:hidden. 

In that case, I think it should not matter if aria-owns is defined or underfined, afaik.